firmware/qemu/src/bin/log.rs

```diff
@@ -339,28 +339,39 @@ fn main() -> ! {
 
     // Tuples
     defmt::info!("empty tuple: {:?}", ());
-    defmt::info!("tuple of ints: {:?}", (1,2,3));
-    defmt::info!("nested tuple of ints: {:?}", (1,2,(3,4,5), (6,7,8)));
-    defmt::info!("crazy nested tuples: {:?}", ((((((((),),),),),),), (((((((), (),),),),),),),));
-    defmt::info!("slice of tuples: {:?}", &[(1,2), (3,4), (5,6)][..]);
-    defmt::info!("tuple of slices: {:?}", (&[1,2,3][..], &[4,5,6][..]));
+    defmt::info!("tuple of ints: {:?}", (1, 2, 3));
+    defmt::info!("nested tuple of ints: {:?}", (1, 2, (3, 4, 5), (6, 7, 8)));
+    defmt::info!(
+        "crazy nested tuples: {:?}",
```

Comment:
while we're at it nitpick for language; I think some users might be uncomfortable with the choice of words here
```suggestion
        "super nested tuples: {:?}",
```